adopt-a-dog/dogs.html

```diff
@@ -0,0 +1,150 @@
+                      nimaxim cum restia volor
+                      reicid ut et etur sunt arum
+                      rendae pla endis re ea erum,
+                      qui doluptae
+                    </p>
+            </div>
+            <div class="grid-dog">
+                <img src="../adopt-a-dog/images/duffy-brook-350225-unsplash.jpg" alt="Duffy">
+              <h5>Duffy</h5>
+                <p>Cost to Adopt: $123.45
+                </p>
+                    <p>Corrum volorit iandae
+                        nimaxim cum restia volor
+                        reicid ut et etur sunt arum
+                        rendae pla endis re ea erum,
+                        qui doluptae
+                    </p>
+            </div>
+            <div class="grid-dog">
+                <img src="../adopt-a-dog/images/tucker-good-589776-unsplash.jpg" alt="Lucas">
+              <h5>Lucas</h5>
+              <p>Cost to Adopt: $123.45
+              </p>
+                  <p>Corrum volorit iandae
+                      nimaxim cum restia volor
+                      reicid ut et etur sunt arum
+                      rendae pla endis re ea erum,
+                      qui doluptae
+                    </p>
+            </div>
+            <div class="grid-dog">
+                <img src="../adopt-a-dog/images/mitchell-orr-237216-unsplash.jpg" alt="Jake">
+              <h5>Jake</h5>
+              <p>Cost to Adopt: $123.45
+              </p>
+                 <p>Corrum volorit iandae
+                      nimaxim cum restia volor
+                      reicid ut et etur sunt arum
+                      rendae pla endis re ea erum,
+                      qui doluptae
+                    </p>
+                    </div>
+            <div class="grid-dog">
+                <img src="../adopt-a-dog/images/jamie-street-562280-unsplash.jpg" alt="Angus">
+              <h5>Angus</h5>
+              <p>Cost to Adopt: $123.45
+              </p>
+                  <p>Corrum volorit iandae
+                      nimaxim cum restia volor
+                      reicid ut et etur sunt arum
+                      rendae pla endis re ea erum,
+                      qui doluptae
+                    </p>
+                    </div>
+            <div class="grid-dog">
+                <img src="../adopt-a-dog/images/alicia-gauthier-253619-unsplash.jpg" alt="Violet">
+              <h5>Violet</h5>
+              <p>Cost to Adopt: $123.45
+              </p> 
+              <p> Corrum volorit iandae
+                      nimaxim cum restia volor
+                      reicid ut et etur sunt arum
+                      rendae pla endis re ea erum,
+                      qui doluptae
+                    </p>
+                    </div>
+                    <div class="see-more-button">
+                        <h3>See More Dogs</h3>
+                      </div>
+          </div>
+      </article>
+
+      <footer class = "main-footer">
```

Comment:
for classes make sure to not leave empty space around the = sign.

<footer class="something">